snap/hooks/configure

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
Aside from when you run snap set, configure runs every time you install and refresh. Maybe some of these are more appropriate for the install and *refresh hook https://snapcraft.io/docs/supported-snap-hooks

Reply:
Good point, I think the folders creation could be limited to the install hook whereas the proc/systemd related conf should instead be added to the actual "start" of the daemon. So that we make sure the configs are applied in the event of a reboot